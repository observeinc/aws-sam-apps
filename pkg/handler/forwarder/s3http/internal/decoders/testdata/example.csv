a,b,c
1,2,
hello,world,nope
"24""",test\help
