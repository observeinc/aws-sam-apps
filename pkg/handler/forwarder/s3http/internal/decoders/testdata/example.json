{
    "hello": "world",
    "anum": 3
}
