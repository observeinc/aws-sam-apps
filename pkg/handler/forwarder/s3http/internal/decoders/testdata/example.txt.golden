{"text":"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor\n"}
{"text":"incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis\n"}
{"text":"nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.\n"}
{"text":"Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore\n"}
{"text":"eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt\n"}
{"text":"in culpa qui officia deserunt mollit anim id est laborum.\n"}
{"text":"\n"}
{"text":"\n"}
